src/core/opentelemetry.ml

```diff
@@ -1007,6 +1016,7 @@ module Trace = struct
     span_id: Span_id.t;
     mutable events: Event.t list;
     mutable attrs: Span.key_value list;
+    mutable links: Span_link.t list;
```

Comment:
I wonder if we should rename to `Link` to be consistent with `Event` (or the other way around) 🤔

Reply:
Ah, good point. I think it's fine, `Link.t` would be a bit ambiguous while `Event.t` is fairly clear, imho. I don't particularly want to just break compatibility for that :-). Maybe some time before 1.0 ?